CMakeLists.txt

```diff
@@ -147,12 +137,13 @@ target_compile_definitions(${PROJECT_NAME}
         $<$<BOOL:${SODIUM_ENABLE_BLOCKING_RANDOM}>:USE_BLOCKING_RANDOM>
         $<$<BOOL:${SODIUM_MINIMAL}>:MINIMAL>
         $<$<C_COMPILER_ID:MSVC>:_CRT_SECURE_NO_WARNINGS>
+        $<$<C_COMPILER_ID:MSVC>:>
```

Comment:
this was committed by mistake, i'll be sure to remove this in the final working commit